tools/SourceKit/tools/sourcekitd/lib/Service/Requests.cpp

```diff
@@ -3995,25 +3996,16 @@ editorFindModuleGroups(StringRef ModuleName, ArrayRef<const char *> Args) {
 
 static bool
 buildRenameLocationsFromDict(const RequestDict &Req,
-                             std::vector<RenameLocations> &RenameLocations,
+                             std::vector<RenameLoc> &RenameLocations,
                              llvm::SmallString<64> &Error) {
   bool Failed = Req.dictionaryArrayApply(KeyRenameLocations,
                                          [&](RequestDict RenameLocation) {
-    int64_t IsFunctionLike = false;
-    if (RenameLocation.getInt64(KeyIsFunctionLike, IsFunctionLike, false)) {
-      Error = "missing key.is_function_like";
-      return true;
-    }
```

Comment:
Have you checked that we send the `()` in all cases that `is_function_like` was set?

Reply:
AFAICT we always send `()` when renaming functions. Xcode definitely sends the parentheses.